app/routes/groups_.$groupId.tsx

```diff
@@ -6,20 +6,23 @@ import { json } from '@remix-run/node';
 import { Button } from '~/components/ui/button';
 
 import { Image } from '~/components/ui/images';
+import { eventsDataPatcher } from '~/modules/events/event';
 
 export const loader: LoaderFunction = async ({ params }) => {
-  const group = await db.group.findFirstOrThrow({ where: { id: params.groupId } });
+  const group = await db.group.findFirstOrThrow({ where: { id: params.groupId }, include: { events: true } });
   return json({ group });
 };
 
 export default function GroupRoute() {
-  const data = useLoaderData<typeof loader>();
+  const rawData = useLoaderData<typeof loader>();
+  const group = rawData.group;
+  const events = eventsDataPatcher(group.events);
   return (
     <div className="bg-primary text-white flex flex-col items-center">
       <div className="md:flex md:flex-col md:max-w-screen-xl">
         <div className="p-2 justify-center md:p-10 md:flex md:flex-row md:border-b-2 md:border-b-white">
           <div className="m-auto rounded-full h-56 w-52  md:h-80 md:w-72 justify-center items-center md:m-8">
-            {!data.group.imgUrl && !data.group.imgAlt ? (
+            {!group.imgUrl && !group.imgAlt ? (
```

Comment:
We could also only conditionally change the `src` and `alt` within the component here:

```tsx
<Image className="w-full" src={group.imgUrl && group.imgAlt ? group.imgUrl : 'https://res.cloudinary.com/dxctpvd8v/image/upload/v1708118888/default-group-photo_xhcpqt.png'} alt={group.imgSrc && group.imgAlt ? group.imgAlt : 'Buddies taking a selfie'} width={288} height={288} />
```